core/src/main/java/net/opentsdb/query/router/RoutingUtils.java

```diff
@@ -230,20 +230,27 @@ public static void rebuildGraph(
         for (int x = 0; x < builder.pushDowns().size(); x++) {
            QueryNodeConfig.Builder pdBuilder = ((QueryNodeConfig) builder.pushDowns()
                    .get(x)).toBuilder();
-           if (x + 1 == builder.pushDowns().size()) {
-             final String pushdownId = ((QueryNodeConfig) builder.pushDowns().get(x)).getId() +
-                     "_" + builder.id();
-             pdBuilder.setResultIds(Lists.newArrayList(
-                     new DefaultQueryResultId(
-                            pushdownId,
-                            builder.dataSource())));
-             pdBuilder.setId(pushdownId);
-           } else {
-             pdBuilder.setResultIds(Lists.newArrayList(
+
+//           if (x + 1 == builder.pushDowns().size()) {
```

Comment:
leave a note about why we commented it out.

Reply:
oh. I might actually put this back.